user_data/strategies/notifier.py

```diff
@@ -1,12 +1,13 @@
 import requests
 import sys
-sys.path.append(Config.EXECUTION_PATH)
+EXECUTION_PATH = '/root/workspace2/execution'
```

Comment:
can we move this to line 1 and give spaces between this and the rest of the imports

Reply:
can we add a comment next to it, saying "do not move this to config", so that in a few years developers won't make the mistake of moving it to config